src/components/donation/Steps/Submit/StripeCheckout/StripeCheckout.tsx

```diff
@@ -41,9 +44,22 @@ export default function StripeCheckout(props: StripeCheckoutStep) {
 
   const dispatch = useSetter();
 
+  const currency = details.currency;
+  const total = +details.amount;
+  const liq = total * (liquidSplitPct / 100);
```

Comment:
Small inconsistency: amounts shown in the "Split donation" step can sometimes differ from amounts shown in the Checkout; this is probably a rounding issue.

Repro steps:
1. Select "Card" donation option
2. Select USD currency
3. Input `4` as amount
4. Click "Continue"
5. Set split to 66/34% (locked/liquid) -> note the amounts:
- Compounded Forever: 2.6400
- Instantly Available: 1.3599
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/7acf374f-8e63-45c5-8023-8a21cefc029c)
6. Click "Continue"
7. Note the amounts differ by $0.01:
- Compounded Forever: 2.63
- Instantly Available: 1.36
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/15434e90-4294-46fa-bc4a-d4a3a1d5e008)

@SovereignAndrey if we're fine with these small rounding inconsistencies, then you can ignore the comment.

Reply:
@misicnenad @ap-justin We do want to ensure that previous values match the donation summary, so as not a cause donors alarm. This is likely a rounding/display issue that would be solved by making sure all displayed values are rounded to:
- 2 decimal places for fiat
- 4 decimal places for crypto

Reply:
found this inconsistency was caused by how `liquid, locked` is determined in splits and checkout  
In splits step:
```js
//get locked first
const locked = total * ( pctLocked  / 100 )
//derive liq
const liq = total - locked
```
while in checkout 
```js
//get liq first
const liq = total * ( pctLiq  / 100 )
//derive locked
const locked = total - liq
```
Nonetheless, still applied 4 decimals for crypto and 2 decimals for fiat @SovereignAndrey